charts/app-of-apps/values-production.yaml

```diff
@@ -0,0 +1,59 @@
+ckanHelmValues:
+  environment: production
+  ckan:
+    args: ["gunicorn --bind 0.0.0.0:5000 wsgi:application --timeout 120"]
+    probes:
+      enabled: true
+    config:
+      site:
+        url: "https://ckan.eks.production.govuk.digital"
```

Comment:
(no action needed) Presumably this will change to `https://ckan.publishing.service.gov.uk/` as part of the switchover?

Reply:
yes, I've added a note to change it on switchover, but for now we're just testing it